lib/db/schema/patch-018-019.sql

```diff
@@ -0,0 +1,199 @@
+
+CREATE TABLE devices (
+  uid BINARY(16) NOT NULL,
+  id INT UNSIGNED NOT NULL,
```

Comment:
We should consider making this smaller than `INT`, especially since it appears in the primary key.  Anyone who registers 4294967295 devices has bigger problems than running out of device id space.